packages/core/src/config/__tests__/__snapshots__/config-resolvers.test.ts.snap

```diff
@@ -0,0 +1,2886 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`resolveApis should not merge recommended extends by default by every level 1`] = `
```

Comment:
These snapshots look not ideal. We should focus one some specific aspect. Why we have plugins and other duplicated thing here?

The concert is we will never review those snapshots because they are unmaintainable. 

We may have a single full snapshots but other tests should focus on the specific area. Not on whole config dump.

Reply:
we reduced the count of snapshots and left only two snapshots for the lint part of the config.